### PR TITLE
Added FPS Command

### PR DESCRIPTION
**Pull request by kaiserandaxl**
# Added FPS Command

**Command to run:** "!fps" or "!framespersecond"
**Admin Level:** Player

Preview:
image

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
#880 vibes 😔

**Comment by kaiserandaxl** [comment]
> #880 vibes 😔

Mine works, idk about yours

**Comment by watameln** [comment]
i tested it says my fps is 0 but i have 60 fps

**Comment by WalkerOfBacon** [comment]
> i tested it says my fps is 0 but i have 60 fps

skill issue

**Comment by kaiserandaxl** [comment]
> i tested it says my fps is 0 but i have 60 fps

Show me

**Comment by watameln** [comment]
go in studio do it yourself i dont have recording utility installed

**Comment by kaiserandaxl** [comment]
> go in studio do it yourself i dont have recording utility installed

A picture will work.

BTW I just tested it, this shows me the FPS

**Comment by watameln** [comment]
test again

**Comment by kaiserandaxl** [comment]
> test again

I've tested like 10 times

**Comment by watameln** [comment]
oh and you have to make renderstepped return 0

**Comment by kaiserandaxl** [comment]
Nothing is wrong

**Comment by watameln** [comment]
https://github.com/Epix-Incorporated/Adonis/assets/90981615/e30c149e-d3b0-4f18-ad72-72f2bbc05395

**Review by WalkerOfBacon — APPROVED** [review]

**Review by SuperCater — APPROVED** [review]

**Comment by Dimenpsyonal** [comment]
Buh.

**Comment by watameln** [comment]
bnuhhhh

File: MainModule/Client/UI/Default/FPS.lua [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			Position = UDim2.new(0, 0, 0, 0);
+			--TextScaled = true;
+			--TextWrapped = true;
+		})
+
+		gTable = window.gTable
+		window:Ready()
+
+		repeat
+			label.Text = `{math.floor(1/game:GetService("RunService").RenderStepped:Wait())} fps`
```
Comment (SuperCater) [inline MainModule/Client/UI/Default/FPS.lua:44]: Using the service table is preferred iirc
```suggestion
			label.Text = `{math.floor(1/service["RunService"].RenderStepped:Wait())} fps`
```

**Review by ccuser44 — COMMENTED** [review]
This is duplicated by Roblox's performance stats feature
And the OS or third party programs implement a better way to do this 2

Also this only shows render FPS but not physics or network FPS

**Comment by Dimenpsyonal** [comment]
> This is duplicated by Roblox's performance stats feature
And the OS or third party programs implement a better way to do this 2

Yeah but the average user doesn't want to have the performance menu open nor do they want to use third party programs

> Also this only shows render FPS but not physics or network FPS

Most people won't know or care what any of those are; they just want a number

**Pull request merged**